alvr/session/src/settings.rs

```diff
@@ -1472,6 +1472,13 @@ pub struct LoggingConfig {
 pub struct SteamvrLauncher {
     #[schema(strings(display_name = "Open and close SteamVR with dashboard"))]
     pub open_close_steamvr_with_dashboard: bool,
+    #[schema(strings(
+        display_name = "SteamVR Quick Launch",
+        help = "Launches SteamVR directly without Steam. This makes launching SteamVR significantly faster, allows SteamVR to be launched offline and avoids the \"app already running\" pop-up."
+    ))]
+    pub quick_launch_steamvr: bool,
+    #[schema(strings(display_name = "SteamVR executable path"))]
+    pub steamvr_executable_path: String,
```

Comment:
I would replace these two settings with `use_steamvr_path: Switch<String>`. The help string can stay the same.